controller/signincontroller.js

```diff
@@ -0,0 +1,25 @@
+
+const UserData = require('../model/signup');
+
+exports.userSignin = function(req,res){
+  UserData.find({email: req.body.email}, function(err, data){
+    console.log(req.body)
+    if(data != null && data != ''){
+      if(req.body.password == data[0].password){
+          res.status(200).json({
+            message:'Login Sucessfully'
+        });
```

Comment:
Encrypt the password and save it on DB. While comparing the password with entered password, compare it by decrypting. Please refer to the Repo which was shared by Venkatesh.